test/integration_test.rb

```diff
@@ -9,9 +9,57 @@ class IntegrationTest < Minitest::Test
 
   def test_allocations
     result = run_vernier({allocation_interval: 1}, "-e", "")
-    assert_valid_firefox_profile(result)
+    assert_valid_firefox_profile(result.to_json)
```

Comment:
It turned out that this test was already broken, as we deserialize the output in `run_vernier`, but this helper expects a JSON string. Added this to fix the test - likely it was silently broken because it never ran as part of the suite before.